### ISSUE TITLE
Using Urban Dictionary search for "@everyone" will cause the bot to mention everyone

### ISSUE THREAD

**Issue by smileytechguy** [cookkkie/mee6#144]
This was recently discovered through a raid on a popular server.  A user repeatedly told the bot to search up `@everyone` on urban dictionary, as fast as they could.  Despite the fact the user could not mention `@everyone`, the bot could, causing everyone on the server to get hundreds of notifications.

A fix for this could sanitize API's responses before the bot sends them or to propose a change to the Discord API which will use a different scheme such as `<@everyone>` or similar.  This could still be abused, and is outside your control, so I suggest the former.

**Comment by KamikazePotato** [comment]
A fix could be removing Admin from Mee6 and removing Mention Everyone

**Comment by smileytechguy** [comment]
True, but I believe most servers would have admin available to bots in general.  I'm not terribly familiar with the bot so it might even have admin features that require the permissions.

Also, it is impossible to fix this in every server.  It'd be better to fix the root cause.

**Comment by KamikazePotato** [comment]
True, though personally I don't see how it would be possible ¯\_(ツ)_/¯

**Comment by smileytechguy** [comment]
Like so I'd think:
(This is psuedocodeish and not even close to a real implementation, just a principle)
```python
def parseAPIResult(in):
    in.title = in.title.replace("@everyone", "@\everyone")
    in.body = in.body.replace("@everyone", "@\everyone")
```

**Comment by matmen** [comment]
just replace @everyone with @\u200beveryone

**Comment by SilBoydens** [comment]
this has already been fixed
